tests/ballerina-integration-test/src/test/resources/testng.xml

```diff
@@ -107,7 +107,7 @@
             <class name="org.ballerinalang.test.auth.AuthzConfigInheritanceTest"/>
             <class name="org.ballerinalang.test.auth.ResourceLevelAuthTest"/>
             <class name="org.ballerinalang.test.auth.ServiceLevelAuthnTest"/>
-            <class name="org.ballerinalang.test.auth.OAuthConfigTest"/>
+            <!--<class name="org.ballerinalang.test.auth.OAuthConfigTest"/>-->
```

Comment:
Shall we remove this as well if not needed?

Reply:
This is needed. This test needs to be refactored to align with the changes introduced in this PR. Created #15049 to track this.